plugin/src/main/java/org/eobjects/build/DotnetBuildMojo.java

```diff
@@ -26,7 +31,17 @@ public void execute() throws MojoExecutionException, MojoFailureException {
                 // restore first if needed
                 helper.executeCommand(subDirectory, "dotnet", "restore");
             }
-            helper.executeCommand(subDirectory, "dotnet", "build", "-c", helper.getBuildConfiguration());
+            final List<String> cmd = new ArrayList<>();
+            cmd.add("dotnet");
+            cmd.add("build");
+            cmd.add("-c");
+            cmd.add(helper.getBuildConfiguration());
+            if(buildFramework != null && !buildFramework.isEmpty())
+            {
```

Comment:
You can save a newline here but add a space after the `if` ;-)

Reply:
fixed